PhotonTheme.Shared/Themes/PhotonTheme.TextBox.xaml

```diff
@@ -91,9 +91,12 @@
         </Setter>
     </Style>
 
-    <!-- Verification TextBox -->
-    <Style x:Key="VerificationBaseTextBox" TargetType="fields:VerificationBox" BasedOn="{StaticResource BaseTextBox}">
-        <Setter Property="VerticalAlignment" Value="Center"/>
+    <!--  Verification TextBox  -->
```

Comment:
Also you can use regions instead comments. I think it would be better, but see for yourself.

```xml
<!-- Region Verification TextBox -->
...
<!-- endregion -->
```

Reply:
That's a good idea, however don't focus too much on that, I am still rethinking how to organize the library. I am leaning towards creating a Base folder and dedicate a separate file for the base style of each control, and then put all the variations in another file, which will be then called by the default dictionary, etc...

But if a control has too many variations I do like the region approach. Thank you!

Reply:
@AlbertAlonso understand you, no problem. 😉